### PR TITLE
Use new fee definition (in ppm)

### PR DESCRIPTION
**Pull request by palango**
As started in raiden-network/raiden-libs#61

### CODE REVIEW

File: pathfinder/tests/fixtures/network_service.py [hunk 1]
```diff
@@ -84,13 +84,13 @@ def channel_descriptions_case_2() -> List:
...
-        (2,  30,  10, 0.1, 5,  40,  15, 0.1),  # capacities  35 --  35
-        (5, 500, 900, 0.1, 4, 750, 950, 0.1),  # capacities 550 -- 700
+        (0, 100,  20, 3000, 1,  50,  10, 3000),  # capacities  90 --  60
+        (1,  40,  10, 2000, 4, 130, 100, 2000),  # capacities 130 --  40
+        (0,  90,  10, 1000, 2,  10,   0, 1000),  # capacities  80 --  10
+        (2,  50,  20, 1500, 3,  50,  20, 1500),  # capacities  50 --  50
+        (3, 100,  40, 1000, 4,  80,   0, 1000),  # capacities  60 -- 120
+        (2,  30,  10, 1000, 5,  40,  15, 1000),  # capacities  35 --  35
+        (5, 500, 900, 1000, 4, 750, 950, 1000),  # capacities 550 -- 700
     ]
```
Comment (err508) [inline pathfinder/tests/fixtures/network_service.py]: In these two blocks the fee refactoring is inconsistent. We currently referenced the variable as `percentage_fee`. Following this, renaming it to `ppm_fee` and a multiplication by 10^4 would keep it consistent.

Comment (palango) [reply pathfinder/tests/fixtures/network_service.py]: Sure it's not consistent, but does it matter!? The numbers are made up anyways.

Comment (err508) [reply pathfinder/tests/fixtures/network_service.py]: No doesn’t matter, but with consistent multiplication tests won't fail !

Comment (palango) [reply pathfinder/tests/fixtures/network_service.py]: How do you mean "test won't fail"?

Comment (err508) [reply pathfinder/tests/fixtures/network_service.py]: We have to recompute the results for every assertion, but anyhow there's different ways to do this correctly. 
I'd still suggest to change percentage_fee to ppm_fee.

**Comment by palango** [comment]
Fixed the fees and changed to `relative_fee` instead of `percentage_fee`

**Pull request merged**